napari/layers/_image_layer/model.py

```diff
@@ -392,12 +365,14 @@ def get_value(self, position, indices):
                 msg = msg + str(value)
             else:
                 msg = msg + f'{value:0.3}'
-        return coord, value, msg
+
+        return msg
 
     def on_mouse_move(self, event):
         """Called whenever mouse moves over canvas.
         """
         if event.pos is None:
             return
-        coord, value, msg = self.get_value(event.pos, self.viewer.dims.indices)
-        self.status = msg
+        self.cursor_position = event.pos
```

Comment:
Again, please note the coordinate space used for each variable.

Reply:
`cursor_position` has now been eliminated, i'll add something everywhere about `event.pos`